<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasuringSystemInputFormClass</class>
 <widget class="QWidget" name="MeasuringSystemInputFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Measuring system input</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>90</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Main contributor name:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="NameInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="DescriptionInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="MainContributorInput">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>DescriptionInput</zorder>
  <zorder>MainContributorInput</zorder>
  <zorder>line</zorder>
  <zorder>continueButton</zorder>
  <zorder>NameInput</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
